### PR TITLE
feat: landing - animate hero

### PR DESCRIPTION
**Pull request by olarclara**
Closes CSB-1862.

## What kind of change does this pull request introduce?

Animates the landing page hero on desktop.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

Verified animations on the following browsers:

1. Chrome
2. Safari
3. Firefox

Performance verification on Chrome.

## Checklist

- [x] Ready to be merged

### CODE REVIEW

**Comment by zehfernandes** [comment]
Awesome work @olarclara! Pretty smooth, and the FPS is not dropping below 30. Thanks for all the effort 🎉👏👏

I have two comments that are important to fix:

1) The Sandpack component is flicking on the screen before the hero. We should prioritize the first paint to be the full hero:
<video src="https://user-images.githubusercontent.com/1891339/143931420-0ee64d99-4c6d-42a9-8961-96dbd66f8274.mp4"></video>




2) If I stop scrolling earlier and edit the code, I can't see my changes. Maybe we can force the swap or force a scrollTo to the position that the editing is fully working. 
<video src="https://user-images.githubusercontent.com/1891339/143931403-6a6c1bc0-39c8-414f-a522-e39c7dd9de17.mp4"></video>

**Comment by olarclara** [comment]
@zehfernandes thank you! the fouc shold be fixed now. I'll take a shot at fixing the latter issue tomorrow, probably in another PR.


https://user-images.githubusercontent.com/24959348/143956633-73407242-48f5-4fa6-b82e-87782b554105.mov

**Review by zehfernandes — APPROVED** [review]

File: website/landing/styles/globalStyles.ts [hunk 1]
```diff
@@ -27,10 +27,56 @@ export const globalStyles = globalCss({
     boxSizing: "border-box",
   },
 
-  body: {
+  // Remove when we replace scrollbar styling in the sandpack library
```
Comment (danilowoz) [inline website/landing/styles/globalStyles.ts:30]: It should have been fixed in another PR, so feel free to remove it

File: website/landing/components/Hero/HeroDesktop/HeroDesktop.tsx [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+      }}
+      template="react"
+    >
+      <AnimatedBox
+        ref={sectionRef}
+        css={{ height: "300vh" }}
+        id="container"
+        style={
+          {
+            "--progress": isMounted ? progress : 0,
```
Comment (danilowoz) [inline website/landing/components/Hero/HeroDesktop/HeroDesktop.tsx:132]: Love this approach

**Review by danilowoz — APPROVED** [review]
Very impressive work @olarclara!

File: website/landing/components/Hero/HeroDesktop/HeroDesktop.tsx (hunk 2 above)
Comment (olarclara) [reply website/landing/components/Hero/HeroDesktop/HeroDesktop.tsx:132]: it was @marcovincit's idea to define all the variables in a single place, it looks super neat and improves perceived performance 🙌🏽

**Pull request merged**